### PR TITLE
Use bech32 MN keys and owner address

### PR DESCRIPTION
**Pull request by monstrobishi**
#### What kind of PR is this?:

/kind fix

#### What this PR does / why we need it:

This modifies setup to use the owner address as this has prevouts to perform "update" functionality, and bech32 keys to support signing updates to poolpairs/oracles via txn-builder

EDIT: Slight modification, see comment below

#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

**Comment by monstrobishi** [comment]
Slight modification, the bech32 address doesn't work for the other setup types, so isolated to oracles for now

File: src/module.playground/setup/setup.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export abstract class PlaygroundSetup<Each> {
    * @return {string} address that should be used for everything
    */
   static get address (): string {
-    return PlaygroundSetup.MN_KEY.operator.address
+    return PlaygroundSetup.MN_KEY.owner.address
```
Comment (fuxingloh) [inline src/module.playground/setup/setup.ts:21]:
```suggestion
    return PlaygroundSetup.MN_KEY.operator.address
```

should revert?

Comment (fuxingloh) [reply src/module.playground/setup/setup.ts:21]: any thoughts about this?

**Comment by fuxingloh** [comment]
do we still need this PR?

**Comment by monstrobishi** [comment]
yes

**Review by fuxingloh — APPROVED** [review]

**Comment by monstrobishi** [comment]
I implementated your suggestion, the rest we still need

**Comment by monstrobishi** [comment]
Slight adjustment

**Comment by fuxingloh** [comment]
should better document it by providing the private key

**Review by fuxingloh — APPROVED** [review]

**Review by canonbrother — APPROVED** [review]

**Pull request merged**